external/storm-hdfs/README.md

```diff
@@ -240,6 +240,23 @@ If you are using Trident and sequence files you can do something like this:
                 .addRotationAction(new MoveFileAction().withDestination("/dest2/"));
 ```
 
+### Data Partitioning
+Data can be partitioned to different HDFS directories based on characteristics of the tuple being processed or purely
+external factors, such as system time.  To partition your your data, write a class that implements the ```Partitioner```
+interface and pass it to the withPartitioner() method of your bolt. The getPartitionPath() method returns a partition 
+path for a given tuple.
+
+Here's an example of a Partitioner that operates on a specific field of data:
+
+```java
+
+    Partitioner partitoner = new Partitioner() {
+            @Override
+            public String getPartitionPath(Tuple tuple) {
+                return Path.SEPARATOR + "city=" + tuple.getStringByField("city");
```

Comment:
The "city=" was from a Hive-specific use case I was working on.  The partitioner would return "city=<name>" to create a partitioning scheme that would be usable for a Hive external table.  I will remove that from the standard documentation and maybe add a hive-specific note.